Engine/EngineConfig.h

```diff
@@ -136,6 +136,23 @@ class EngineConfig {
 
     bool is_underwater = false;
     bool is_targeting = false;
+
+    bool no_grab = false;
```

Comment:
In windowed mode would recommend defaulting to no_grab = true, window_x = -1 and window_y = -1.

Reply:
Agreed about window_x/window_y. But don't know about no_grab. Usually enabled grab is what player wants and what is common default in other games. For developing yes, it is a bit annoying, thats why there is override commandline switch.

Reply:
I just pushed some changes around screen mode change and changed window_x/window_y defaults. If you still believe no_grab default should be changed as well let me know.